infrastructure/metadata/infrastructure/testdriver/bless.html.ini

```diff
@@ -0,0 +1,31 @@
+[bless.html]
```

Comment:
@LukeZielinski as an FYI, this expectation file is probably the best example so far of something that's time-consuming to write manually. I had to copy the section the right number of times and copy all the names from log output until I got it right...

Reply:
Context: https://github.com/web-platform-tests/wpt/issues/12697, and it's easier to update expectations to match actual results in Chromium, or at least I know how to do it there.